plugins/pdr_deterministic_plugin/build/docker_build.sh

```diff
@@ -100,6 +100,8 @@ pushd ${SCRIPT_DIR}
 BUILD_DIR=$(create_out_dir)
 
 cp Dockerfile ${BUILD_DIR}
+curl -l -k 'https://l-webdev01:28443/raw/golan_fw.git/master_rc/docs!Field_BER_Thresholds.csv' -o Field_BER_Thresholds.csv
```

Comment:
is there an external github repo that can be used? if the server is down, we won't be able to create the PDR plugin image...